requirements-dev.txt

```diff
@@ -31,7 +31,7 @@ nbsphinx
 numexpr>=2.6.8
 openpyxl
 pyarrow>=0.9.0
-tables>=3.4.2
+tables>=3.5.2
```

Comment:
i don't think this would work because it would require a newer numpy than is allowed in this version (1.13)